src/Tasks/Microsoft.NET.Build.Tasks/CacheArtifactParser.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml;
+using System.Globalization;
+using System.Xml.Linq;
+using NuGet.Versioning;
+using NuGet.Packaging.Core;
+
+#if PRODUCT
+using Microsoft.Build.Framework;
+#endif
+
+namespace Microsoft.NET.Build.Tasks
+{
+    //<!-- The following provides the logic to parse the output - artifact.xml -->
```

Comment:
I think you misunderstood my previous ask.

In C#, class summary comments should look like the following:

```
    /// <summary>
    /// Represents a single diagnostic message, such as a compilation error or a project.json parsing error.
    /// </summary>
    internal class DiagnosticMessage
```